libcodechecker/package_context.py

```diff
@@ -136,8 +134,8 @@ def __populate_analyzers(self):
             # will be checked later.
```

Comment:
I think we can remove this checking. The supported analyzers list should be configured in the pckg_layout json always and we should load the names from there.